Marlin/Voxel8_I2C_Commands.cpp

```diff
@@ -292,6 +290,43 @@ void I2C__GetFirmwareVersion(uint8_t cartridge) {
 }
 
 /**
+ * Read the voltage sense pin on a cartridge and print it on the serial port.
+ * On the pneumatics cartridge, this should correspond to the solenoid being 
+ * active. For FFF, this represents if the hot end is active.
+ * @parameter cartridge           Address of the target (cartridge)
+ */
+void I2C__GetVoltageSense(uint8_t cartridge) {
```

Comment:
This is a little bit of a misnomer, it should really be the i2c address and not the cartridge as many of these also address the cartridge holder. The convention at this point is that the M-code passes the correct I2C address, which simplifies these a bit